samples/excel/45-named-item/create-and-use-named-item-for-range.yaml

```diff
@@ -0,0 +1,180 @@
+id: excel-create-and-use-named-range-item
+name: Create and use named range item
+description: Create and use a named range item
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |-
+        $("#add-name").click(() => tryCatch(addName));
+        $("#format-named-range").click(() => tryCatch(formatRangeByName));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function addName() {
+            await Excel.run(async (context) => {
+
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const headerRange = sheet.getRange("A1:E1");
+
+                sheet.names.add("ExpensesHeader", headerRange);
+
+                const namedItems = sheet.names.load();
+
+                await context.sync();
+
+                console.log("This workbook contains " + namedItems.items.length + " named items.");
+
+                for (let i = 0; i < namedItems.items.length; i++) {
+                    console.log(JSON.stringify(namedItems.items[i])) + "\n";
```

Comment:
What is the + "\n" doing outside the console log (and you don't need it).
Also, why the JSON.stringify here?